packages/react-slider/src/components/Slider/useSliderState.tsx

```diff
@@ -1,244 +1,68 @@
 import * as React from 'react';
+import { clamp, useControllableState, useEventCallback } from '@fluentui/react-utilities';
 import { useFluent } from '@fluentui/react-shared-contexts';
 import {
-  clamp,
-  useBoolean,
-  useControllableState,
-  useEventCallback,
-  useUnmount,
-  useMergedRefs,
-} from '@fluentui/react-utilities';
-import {
-  calculateSteps,
-  getKeydownValue,
-  getMarkPercent,
-  getMarkValue,
-  getPercent,
-  on,
-  renderMarks,
-} from '../../utils/index';
+  railOffsetVar,
+  railStepsPercentVar,
+  railProgressVar,
+  thumbPositionVar,
+  railDirectionVar,
+} from './useSliderStyles';
 import type { SliderState } from './Slider.types';
 
-// TODO: Awaiting animation time from design spec.
-export const animationTime = '0.1s';
+const getPercent = (value: number, min: number, max: number) => {
```

Comment:
You'll probably want to make this into a shared utility once you make `RangedSlider` but we can keep it here until then.

Reply:
yeah, it was originally in a utility folder, but it ended up being the ONLY thing there. Seemed an unnecessary folder/file/export for one small function